src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js

```diff
@@ -2,13 +2,36 @@ import dayjs from 'dayjs';
 import {
   filterOnOrAfterDate,
   filterBeforeDate,
+  joinOrConditions,
+  joinAndConditions,
 } from '../resource-filter-builder/resourceFilterBuilder';
 
-export const filterTimeEntriesOnDate = (date) => {
-  const startOfDateFilter = filterOnOrAfterDate('actualStartTime', date);
+export const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
```

Comment:
we no longer need to export this function as `buildTimeCardFilter` is the one used by `TimeCard` now

Reply:
export removed.